include/mbedtls/ctr_drbg.h

```diff
@@ -234,6 +274,11 @@ void mbedtls_ctr_drbg_set_prediction_resistance( mbedtls_ctr_drbg_context *ctx,
  *                      seed or reseed. The default value is
  *                      #MBEDTLS_CTR_DRBG_ENTROPY_LEN.
  *
+ * \note                For compliance with NIST SP 800-57A, the entropy length must be least
```

Comment:
Security strength is measured in bits, `len` is length in bytes, this might be confusing. We could clarify it by writing "\p len * 8" or "bit length" instead of length.